docs/gallery/gallery/scales/prescaled_data.jl

```diff
@@ -6,6 +6,7 @@
 # ---
 
 using AlgebraOfGraphics, CairoMakie
+using CairoMakie.Makie
```

Comment:
Maybe this is not needed? I think `using CairoMakie` already brings `Makie` in scope.